sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-program-and-offering-status-report.sql

```diff
@@ -0,0 +1,120 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Program_And_Offering_Status_Report',
+    '(
+        select 
+          il.name as "Institution Location Name",
+          ep.program_name as "Program Name",
+          ep.sabc_code as "SABC Program Code",
+          ep.regulatory_body as "Regulatory Body",
+          ep.credential_type as "Credential Type",
+          ep.completion_years as "Program Length",
+          case
+            when (ep.delivered_on_site = true and ep.delivered_online = true) then ''Blended''
+            when ep.delivered_on_site = true then ''On-site''
+            when ep.delivered_online = true then ''Online''
+          end as "Delivery",
+          ep.course_load_calculation as "Credit or Hours",
+          ep.program_status as "Program Status",
+          case 
+            when ep.is_active = false then ''True''
+            when ep.is_active = true then ''False''
+          end as "Program Deactivated?",
+          ep.effective_end_date as "Program Expiry Date",
+          epo.offering_name as "Offering Name", 
+          to_char(
+            epo.study_start_date,
+           ''YYYY-MM-DD''
+          ) as "Study Start Date",
+          to_char(
+            epo.study_end_date,
+           ''YYYY-MM-DD''
+          ) as "Study End Date",
```

Comment:
No conversion needed for `study_start_date` and `study_end_date` since they are date only columns and should be returned as expected.
For other reports it was done as below (e.g. ECert_Errors_Report).
```sql
CAST(education_programs_offerings.study_start_date AS varchar) as "Study Start Date",
CAST(education_programs_offerings.study_end_date AS varchar) as "Study End Date",
```